public/messages.html

```
@@ -21,23 +21,89 @@ <h1 class="text-xl font-bold flex items-center text-gray-800 dark:text-gray-100"
+        <div class="grid grid-cols-1 md:grid-cols-2 gap-6 mb-8">
+          <!-- Send New Message Request -->
+          <div class="bg-white dark:bg-gray-800 rounded-2xl p-5 border border-gray-200 dark:border-gray-700 shadow-sm">
+            <h3 class="text-lg font-bold text-gray-900 dark:text-gray-100 mb-2 flex items-center">
+              <i class="fa-solid fa-paper-plane text-teal-500 mr-2"></i>New Message Request
+            </h3>
+            <p class="text-xs text-gray-500 dark:text-gray-400 mb-4">
+              Enter the exact email address of the recipient to send a privacy-preserving message request.
+            </p>
+            <form id="send-request-form" class="space-y-3">
+              <div>
+                <input type="email" id="request-email" placeholder="recipient@example.com" required autocomplete="off" class="w-full px-4 py-2 text-sm rounded-lg border border-gray-300 dark:border-gray-600 bg-gray-50 dark:bg-gray-900 text-gray-900 dark:text-gray-100 focus:ring-2 focus:ring-teal-500 focus:outline-none">
```

HTMLHint (1.9.2)

[warning] No matching [ label ] tag found.

(input-requires-label)